frontend/src/components/ViewTrips/edit-trip-button.js

```diff
@@ -4,19 +4,15 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import Button from 'react-bootstrap/Button';
 
 /**
- * @typedef { DefaultFormData } FormattedTripData
- * {@link FormattedTripData} is an alias for the `typedef`
- * {@link DefaultFormData} defined in `ViewTrips/save-trip-modal.js`.
+ * {@link TripData} defined originally in `ViewTrips/trip.js`.
```

Comment:
I believe you don't need to link it here, but ok

Reply:
Yeah I have been doing that across all my files where I use a typedef that wasn't defined in the file for reference (if they want to find that type definition in the editor).